### DISCUSSION TITLE
How can I programmatically changes the currently selected item/index of a list?

### DISCUSSION THREAD

**Discussion by refaelsh** [jtdaugherty/brick#473] [Q&A] [answered]
How can I programmatically changes the currently selected item/index?
I don't see anything in here: https://hackage.haskell.org/package/brick-1.9/docs/Brick-Widgets-List.html.

Please help :-)

**Comment by jtdaugherty** [comment, marked as answer]
You can you use [listMoveTo](https://hackage.haskell.org/package/brick-1.9/docs/Brick-Widgets-List.html#v:listMoveTo).

**Reply by refaelsh** [reply]
Yeap. Somehow missed it in the docs. Thank you!
